unstructured/partition/lang.py

```diff
@@ -167,34 +167,68 @@ def prepare_languages_for_tesseract(languages: Optional[List[str]] = ["eng"]):
     return TESSERACT_LANGUAGES_SPLITTER.join(converted_languages)
 
 
-def check_languages(languages: Optional[List[str]], ocr_languages: Optional[str]):
-    """Handle `ocr_languages` and `languages`, defining `languages` to ['eng'] as default and
-    converting `ocr_languages` if needed"""
-    if languages is None:
-        languages = ["eng"]
+def check_language_args(
+    languages: Optional[List[str]], ocr_languages: Optional[str]
+) -> Union[list[str], None]:
+    """Handle users defining both `ocr_languages` and `languages`, giving preference to `languages`
+    and converting `ocr_languages` if needed, but defaulting to `None.
+
+    `ocr_languages` is only a parameter for `auto.partition`, `partition_image`, & `partition_pdf`.
+    `ocr_languages` should not be defined as 'auto' since 'auto' is intended for language detection
+    which is not supported by `partition_image` or `partition_pdf`."""
+    # --- Clean and update defaults
+    if ocr_languages == "auto":
+        raise ValueError(
+            "`ocr_languages` is deprecated but was used to extract text from pdfs and images. "
+            "The 'auto' argument is only for language *detection* when it is assigned "
+            "to `languages` and partitioning documents other than pdfs or images. "
+            "Language detection is not currently supported in pdfs or images."
```

Comment:
When specifying multiline strings like this, I make it my habit to place the space separating each line at the _beginning_ of the second and later lines, like:
```python
"ocr_languages is deprecated but was used to extract text from pdfs and images."
" The 'auto' argument is only for language *detection* when it is assigned"
" to `languages` and partitioning documents other than pdfs or images."
" Language detection is not currently supported in pdfs or images."
```

That way its presence can be confirmed by quickly scanning down the aligned left side rather than the ragged right side, which is much more work for the eye. This leads to fewer merged words in exception output, which is a perennial problem.